contracts/staking/TokenStaking.sol

```diff
@@ -108,7 +101,6 @@ contract TokenStaking is Initializable, IStaking, Checkpoints {
     uint256 public slashingQueueIndex;
 
     event Staked(
-        StakeType indexed stakeType,
```

Comment:
I'm afraid we can't drop this. There's already a bunch of external apps (e.g., Dune queries) that consume events in their current form.

Reply:
Got it, added back ✔️